PoshBot/Classes/ScheduledMessage.ps1

```diff
@@ -77,40 +77,25 @@ class ScheduledMessage {
     }
 
     [bool]HasElapsed() {
-        if ($this.Stopwatch.ElapsedMilliseconds -gt $this.IntervalMS) {
-            $now = (Get-Date).ToUniversalTime()
-            if ($now -ge $this.StartAfter) {
-                $this.TimesExecuted += 1
-                return $true
-            } else{
-                return $false
-            }
+        $now = (Get-Date).ToUniversalTime()
+        if ($now -gt $this.StartAfter) {
+            $this.TimesExecuted += 1
+            return $true
         } else {
             return $false
         }
     }
 
     [void]Enable() {
         $this.Enabled = $true
-        $this.StartTimer()
     }
 
     [void]Disable() {
         $this.Enabled = $false
-        $this.StopTimer()
-    }
-
-    [void]StartTimer() {
-        $this.Stopwatch.Start()
-    }
-
-    [void]StopTimer() {
-        $this.Stopwatch.Stop()
     }
 
-    [void]ResetTimer() {
-        $this.Stopwatch.Reset()
-        $this.Stopwatch.Start()
```

Comment:
Looks like the stopwatch is not longer need. Can you remove it from the class?

Reply:
Good catch. I removed it from the scheduler class.  I did a search and the only other areas the stopwatch seems to be referenced were in the actual Bot instance for tracking uptime.

Let me know if you find anything else that requires changing!